actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -252,6 +252,10 @@ def require(key)
     # Alias of #require.
     alias :required :require
 
+    def require_all(keys)
```

Comment:
Not sure actually, but the same problem would then apply to `require_all` as well.

Typically the pattern is to use `require` for parent hashes, and `permit` for individual attributes. An individual attribute requirement is usually checked by model validation.